### PR TITLE
Add "WORKS" and "ABOUT_ME" sections to the portfolio.

### PR DESCRIPTION
**Pull request by M-Anwar-Hussaini**
Hi dear Code Reviewers.

In this PR I tried my best to apply all the requirements needed for this two sections.
If there is any kind of issues on it, please do mention me in a descriptive expressions  and recommend the best suggestions.

Thanks a lot.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,6 +81,209 @@ <h1 class="heading-primary">
...
+        <!-- CARDS -->
+        <div class="cards">
+          <!-- Languages card -->
+          <div class="card">
+            <img
+              src="imgs/card-icons/icon-languages.png"
+              alt=""
+              class="card-icon"
+            />
+            <h3 class="heading-tertiary">Languages</h3>
```
Comment (Kossi-stack) [inline index.html:236]: - [ ] Kindly put the technology categories ("Languages", "Frameworks" and "Skills") in a ul, each of them in a li

File: index.html [hunk 2]
```diff
@@ -81,6 +81,209 @@ <h1 class="heading-primary">
...
+
+      <section class="section-about-me">
+        <div class="about-me-header">
+          <h2 class="heading-secondary">About me</h2>
+          <p class="about-me-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <a href="#" class="btn">Get my resume</a>
```
Comment (Kossi-stack) [inline index.html:214]: - [ ] Kindly follow the button styling: Active, Hover, and Pressed 
- Figma Design
Screenshot 2023-04-27 at 23 28 33

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @M-Anwar-Hussaini,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 3]
```diff
@@ -81,6 +81,209 @@ <h1 class="heading-primary">
...
+          </li>
+          <!-- Framworks card -->
+          <li class="card">
+            <img
+              src="imgs/card-icons/icon-frameworks.png"
+              alt="Frameworks Icon"
+              class="card-icon"
+            />
+            <h3 class="heading-tertiary">Frameworks</h3>
+            <div class="card-items-conainer">
```
Comment (ademibowale) [inline index.html:252]: - [ ] Thank you so much for implementing changes requested from your previous reviewer that looks so good,but i observed some broken images on your UI ,Kindly fix this 
- [ ] I also notice that your background images not display too please kindly fix it to make  you website look exactly like original design on figma template.
- [ ] your design
figmwww
- [ ] Your design 
figm ww
   - [ ] Figma design
figma templ3

**Review by ademibowale — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @M-Anwar-Hussaini     😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

just a little bit

## Highlights
  ✅ Nice user interface
  ✅Correct use of Github Flow
  ✅ No linter errors
✅ Descriptive Pull Request
  

## Required Changes ♻️




_Check the comments under the review._




## Optional suggestions 
   
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (M-Anwar-Hussaini) [reply index.html:252]: Hey @ademibowale.

I think there might be some problems with yours side. for me , it works perfect.

Could You please re-check it. @ademibowale .

Comment (M-Anwar-Hussaini) [reply index.html:252]: image
image
image
image

Comment (M-Anwar-Hussaini) [reply index.html:252]: It's the "add-works-and-aboutme-sections" branch.

**Review by V-Blaze — APPROVED** [review]
Hi @M-Anwar-Hussaini  

# Approved 🍏 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
green-lantern-ok

## Highlights
- Clean and well-organized code✔✔
- Great communication with the reviewers :heavy_check_mark: 
- Project looks like the Figma design :heavy_check_mark: 

## Optional suggestions

- [OPTIONAL] Great job using the correct license in your project. :+1: :+1: But I noticed you haven't customized it to match your project. Please go through License file and make it more professional :+1: :smile: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hello @M-Anwar-Hussaini 👋,

_This is an additional review as a result of the second opinion request._

I confirm that all images are displayed on the portfolio page.  @ademibowale missed testing the page on a live server; that's why the images are not displayed on the page.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

[OPTIONAL SUGGESTION]
Based on the project requirements,  mobile version should be responsive until the desktop screen size.. The previous reviewers overlooked this requirement, but I strongly recommend addressing it in order to ensure that your project meets all the necessary requirements

Keep rocking 🚀!

**Pull request merged**